resources/views/auth/users/show.blade.php

```diff
@@ -46,7 +46,9 @@
     <div class="card-body tab-content" id="pills-tabContent">
         <div class="tab-pane fade show active card-text" id="account" role="tabpanel">
 
-            <form method="post" action="{{route('account.update')}}">
+            <p>{{ Route::currentRouteName() }}</p>
```

Comment:
Check this as it currently resolves to `passport.tokens.index` which I'm guessing is not what is wanted

Reply:
Resolved in https://github.com/MultiCrew/copilot/pull/96/commits/89320b3ffecef5df4f302f151ecddcdb796b4d8b

Reply:
That commit doesn't show any changes in users/show

Reply:
Oops, that was a leftover debugging output (now removed in https://github.com/MultiCrew/copilot/commit/0f1e6427335842e14ab26b8171d1dce855d17699). The actual active nav element logic was fixed to reflect the erroneous route name in https://github.com/MultiCrew/copilot/commit/89320b3ffecef5df4f302f151ecddcdb796b4d8b